apps/web/components/getting-started/steps-views/UserSettings.tsx

```diff
@@ -1,120 +1,178 @@
-import { zodResolver } from "@hookform/resolvers/zod";
-import { useEffect } from "react";
-import { useForm } from "react-hook-form";
-import { z } from "zod";
+import { useRouter } from "next/navigation";
+import type { FormEvent } from "react";
+import { useEffect, useState } from "react";
 
-import dayjs from "@calcom/dayjs";
-import { useTimePreferences } from "@calcom/features/bookings/lib";
-import { FULL_NAME_LENGTH_MAX_LIMIT } from "@calcom/lib/constants";
+import { DEFAULT_SCHEDULE } from "@calcom/lib/availability";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { telemetryEventTypes, useTelemetry } from "@calcom/lib/telemetry";
+import { IdentityProvider } from "@calcom/prisma/enums";
 import { trpc } from "@calcom/trpc/react";
-import { Button, TimezoneSelect, Icon, Input } from "@calcom/ui";
+import { Button, showToast, Icon, Tooltip, RadioGroup } from "@calcom/ui";
 
-import { UsernameAvailabilityField } from "@components/ui/UsernameAvailability";
+type AccountOption = "personal" | "team" | "org";
```

Comment:
"org" type cannot be selected in the current onboarding design. It has a tooltip saying "Contact sales"